pkg/applylib/applyset/applyset_test.go

```diff
@@ -1,128 +0,0 @@
-/*
```

Comment:
Do we want to delete the test?  If the test doesn't pass I'd rather t.Skip() it with a reason, and then we can try to make our mocks better?  I'm trying to incorporate mockkubeapiserver (and applyset) into kubebuilder-declarative-pattern here https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/pull/248 .  I can keep copying and pasting back and forth, but I'm also trying to establish a go.mod there as that might be a good place to maintain it (eventually!)